library/include/rocblas-functions.h

```diff
@@ -9075,58 +9075,65 @@ ROCBLAS_EXPORT rocblas_status rocblas_zsyr2_strided_batched(rocblas_handle
  *    level 3 BLAS
  * ===========================================================================
  */
+
 /*! \brief BLAS Level 3 API
 
     \details
 
-    herk performs one of the matrix-matrix operations for a Hermitian rank-k update
-
-    C := alpha*op( A )*op( A )^H + beta*C
+    hemm performs one of the matrix-matrix operations:
 
-    where  alpha and beta are scalars, op(A) is an n by k matrix, and
-    C is a n x n Hermitian matrix stored as either upper or lower.
+    C := alpha*A*B + beta*C if side == rocblas_side_left,
+    C := alpha*B*A + beta*C if side == rocblas_side_right,
 
-        op( A ) = A, and A is n by k if transA == rocblas_operation_none
-        op( A ) = A^H and A is k by n if transA == rocblas_operation_conjugate_transpose
+    where alpha and beta are scalars, B and C are m by n matrices, and
+    A is a Hermitian matrix stored as either upper or lower.
 
     @param[in]
     handle    [rocblas_handle]
               handle to the rocblas library context queue.
 
     @param[in]
-    uplo    [rocblas_fill]
-            rocblas_fill_upper:  C is an upper triangular matrix
-            rocblas_fill_lower:  C is a  lower triangular matrix
+    side  [rocblas_side]
+            rocblas_side_left:      C := alpha*A*B + beta*C
+            rocblas_side_right:     C := alpha*B*A + beta*C
 
     @param[in]
-    transA  [rocblas_operation]
-            rocblas_operation_conjugate_transpose:  op(A) = A^H
-            rocblas_operation_none:                 op(A) = A
+    uplo    [rocblas_fill]
+            rocblas_fill_upper:  A is an upper triangular matrix
+            rocblas_fill_lower:  A is a  lower triangular matrix
 
     @param[in]
-    n       [rocblas_int]
-            n specifies the number of rows and columns of C. n >= 0.
+    m       [rocblas_int]
+            m specifies the number of rows of B and C. m >= 0.
 
     @param[in]
-    k       [rocblas_int]
-            k specifies the number of columns of op(A). k >= 0.
+    n       [rocblas_int]
+            n specifies the number of columns of B and C. n >= 0.
 
     @param[in]
     alpha
             alpha specifies the scalar alpha. When alpha is
-            zero then A is not referenced and A need not be set before
-            entry.
+            zero then A and B are not referenced.
 
     @param[in]
     A       pointer storing matrix A on the GPU.
-            Martrix dimension is ( lda, k ) when if transA = rocblas_operation_none, otherwise (lda, n)
+            A is m by m if side == rocblas_side_left
+            A is n by n if side == rocblas_side_right
```

Comment:
Maybe mention that the imaginary parts of the diagonal need not be set (as is done in lapack).